notebooks/01_data_preprocessing.ipynb

```diff
@@ -22,7 +22,8 @@
     "import scipy as sp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    rda_scratch       = "/gpfs/csfs1/collections/rda/scratch/harshah"
</pre><p>Please do not use your scratch directory. Others won't have write access to this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ERA5-dashboard-cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>